client/src/actions/users.js

```diff
@@ -7,10 +7,10 @@ import { ADD_USER_DATA } from "./types";
 /**
  * This action creator will be called when a user registers.
  *
- * @param userData object containing user email, given and family name and password
+ * @param userData object containing user email, team and organization name and password
  * @returns a thunk responsible for posting the data to the api and dispatching
  */
-export const addUserAction = (userData) => async (dispatch) => {
+export const addUserAction = (userData, apiResult) => async (dispatch) => {
```

Comment:
uhm.. what to do with this apiResult?

Reply:
I think this method is not called yet: 
![image](https://user-images.githubusercontent.com/17421174/124903761-a0241e00-e027-11eb-89ff-8448e7fc6070.png)
(it got commented as shown image above)

Reply:
The api result was added in the commit in thunder which I just cherry picked, and the function is not being used because we do not create a User model anymore when registering, we just create a Team model.

Reply:
The only reason I left the adding of `apiResult` in this function while cherry picking was in case we go back to using User model, don't want to loose effort